src/main/java/com/jagrosh/jmusicbot/JMusicBot.java

```diff
@@ -99,7 +99,7 @@ public static void main(String[] args)
                         new PingCommand(),
                         new SettingsCmd(bot),
                         
-                        new LyricsCmd(bot),
+                        // new LyricsCmd(bot),
```

Comment:
This line was causing problems when I tried to run the bot from a jar executable.